cross/mono/PLIST

```diff
@@ -1,94 +1,73 @@
-rsc:bin/al
-rsc:bin/al2
-rsc:bin/csc
-rsc:bin/csc-dim
-rsc:bin/csi
-rsc:bin/gacutil
-rsc:bin/gacutil2
-rsc:bin/ilasm
-rsc:bin/makecert
-rsc:bin/mcs
-rsc:bin/mdassembler
-rsc:bin/mdoc-assemble
-rsc:bin/mdoc-export-html
-rsc:bin/mdoc-export-msxdoc
-rsc:bin/mdoc-update
-rsc:bin/mdoc-validate
-rsc:bin/mdvalidater
-rsc:bin/mkbundle
+bin:bin/al
+bin:bin/al2
+bin:bin/csc
+bin:bin/csi
+bin:bin/gacutil
+bin:bin/gacutil2
+bin:bin/ilasm
+bin:bin/makecert
+bin:bin/mcs
+bin:bin/mdassembler
+bin:bin/mdoc-assemble
+bin:bin/mdoc-export-html
+bin:bin/mdoc-export-msxdoc
+bin:bin/mdoc-update
+bin:bin/mdoc-validate
+bin:bin/mdvalidater
+bin:bin/mkbundle
```

Comment:
please keep files as `rsc` as it is not possible to _strip_  those (and other) files.

Reply:
Ok I have fixed the bin as rsc